schema/home/home_page_artwork_module.js

```diff
@@ -64,14 +64,32 @@ export function createHomePageArtworkModule(moduleTypeName) {
       id: {
         type: GraphQLString,
         description: 'ID of generic gene rail to target',
+        deprecationReason: 'Favor more specific `generic_gene_id`',
+      },
+      generic_gene_id: {
+        type: GraphQLString,
+        description: 'ID of generic gene rail to target',
+        deprecationReason: 'Favor more specific `generic_gene_id`',
+      },
+      followed_artist_id: {
+        type: GraphQLString,
+        description: 'ID of followed artist to target for related artist rails',
+      },
+      related_artist_id: {
+        type: GraphQLString,
+        description: 'ID of related artist to target for related artist rails',
       },
     },
-    resolve: (root, { key, id }) => {
+    resolve: (root, { key, id, followed_artist_id, related_artist_id }) => {
       // is id a generic gene?
-      const params = find(genericGenes, ['id', id]);
+      let params = find(genericGenes, ['id', id]);
```

Comment:
So the `followed_artist_id` and `related_artist_id` arguments are passed to each invocation, but not in all cases do they actually apply (but could they be e.g. a generic gene) ?